homeassistant/components/here_travel_time/__init__.py

```diff
@@ -38,6 +41,23 @@ async def _async_update_at_start(_: HomeAssistant) -> None:
     return True
 
 
+def alert_for_multiple_entries(hass: HomeAssistant) -> None:
+    """Check if there are multiple entries for the same API key."""
+    if len(hass.config_entries.async_entries("here_travel_time")) > 1:
```

Comment:
Use the DOMAIN const here
```suggestion
    if len(hass.config_entries.async_entries(DOMAIN)) > 1:
```

Reply:
Thanks!